app/services/doc_auth/response.rb

```diff
@@ -47,7 +48,12 @@ def first_error_message
     end
 
     def attention_with_barcode?
-      raise NotImplementedError
+      false
+    end
+
+    def ocr_pii
+      return unless attention_with_barcode?
+      pii_from_doc.slice(:first_name, :last_name, :dob)
```

Comment:
we also get the address from OCR, right?

Reply:
For the designs, these are the values we're asking the user to confirm, since they're the ones that the user cannot change once they get to the "Verify" step in the flow. Though, now that you raise it, maybe there's a clearer name to describe them.

Reply:
Maybe it would help to move this method out of this class and plug the logic directly into `ImageUploadResponsePresenter`, at least to avoid the confusion of what "OCR PII" means in relation to the base response class.

Reply:
moving out of this class makes sense to me I think, especialy if it's just relevant to that one API endpoint